architecture/routing.adoc

```diff
@@ -107,24 +107,53 @@ file.
 
 ====
 
-IMPORTANT: While both secure and unsecure routes are possible with an OpenShift
-instance, Red Hat recommends a secure routing service.
+IMPORTANT: While both secure and unsecure routes are possible with an OpenShift instance, Red Hat recommends a secure routing service.
 
 == Path Based Routes
-Path based routes are routes that are specifying a path component that can be
-compared against a URL. This implies that the traffic for the route is HTTP
-based. Routers should match routes based on the most specific path to the least
-however this is implementation dependent.  For example:
-
-1.  Route with www.example.com/test
-  * www.example.com/test should be accessible
-  * www.example.com should not be accessible
-1.  Routes with www.example.com/test and www.example.com
-  * www.example.com/test should be accessible
-  * www.example.com should be accessible
-1.  Route with www.example.com
-  * www.example.com/test should be accessible (matched by the host, not the route)
-  * www.example.com should be accessible
+Path based routes specify a path component that can be compared against a URL. This implies that the traffic for the route is HTTP based. Routers should match routes based on the most specific path to the least. However, this depends on your implementation. For example:
+
+////
+*  For a route with \_www.example.com/test_:
+** \_www.example.com/test_ should be accessible
+** \_www.example.com_ should not be accessible
+*  For routes with \_www.example.com/test_ and \_www.example.com_:
+** \_www.example.com/test_ should be accessible
+** \_www.example.com_ should be accessible
+*  For routes with \_www.example.com_:
+** \_www.example.com/test_ should be accessible (matched by the host, not the route)
+** \_www.example.com_ should be accessible
+////
+
+[cols="3*"]
+|===
+|Route
+|When Compared to
+|Accessibility
```

Comment:
Changed to "Accessible". 

I tossed up whether "Should be Accessible" would be better, or even "Is it Accessible?" . Thoughts?